src/test/scala/uk/gov/homeoffice/rtemailer/emailsender/GovNotifyEmailSenderSpec.scala

```diff
@@ -139,8 +141,13 @@ class govNotifyEmailSenderSpec extends CatsEffectSuite {
   val parentCaseId = new ObjectId().toHexString()
   val defaultCaseId = new ObjectId().toHexString()
 
-  val fakeMongoWrapper = new GovNotifyMongoWrapper()(testAppContext) {
-    override def caseObjectFromEmail(email :Email) :IO[Either[GovNotifyError, Option[MongoDBObject]]] = {
+  val fakeDatabase = new Database {
+    override def name() = "FakeDB"
+    override def obtainLock() :IO[Lock] = ???
```

Comment:
No implementions

Reply:
No implementations because the test isn't testing that yet. I have a tickets DPSPS-1430 and DPSPS-1431 to test the locking mechanism properly.